bytes/json.go

```diff
@@ -20,7 +20,7 @@ func unquoteBytes(s []byte) (t []byte) { //nolint:gocyclo // It's OK.
 	r := 0
 	for r < len(s) {
 		c := s[r]
-		if c == '\\' || c == '"' || c < ' ' {
```

Comment:
You need to write in the comments to the function that it has been modified after copying it from the encoding/json package

Reply:
An appropriate comment was added to both functions.